### PR TITLE
feat(daemon): rolling update via client

### PR DESCRIPTION
**Pull request by deepankarm**

### CODE REVIEW

File: daemon/clients/pods.py [hunk 1]
```diff
@@ -8,6 +16,35 @@ class AsyncPodClient(AsyncPeaClient):
...
+        async with aiohttp.request(
+            method='PUT',
+            url=f'{self.store_api}/{daemonize(id, self._kind)}',
+            params={
+                'kind': UpdateOperation.ROLLING_UPDATE.value,
+                'dump_path': dump_path,
+            },
+        ) as response:
+            response_json = await response.json()
+            if response.status != HTTPStatus.CREATED:
```
Comment (jacobowitz) [inline daemon/clients/pods.py:40]: Shouldnt this rather be HTTPStatus.OK?

**Review by jacobowitz — CHANGES_REQUESTED** [review]
only 1 comment regarding the HTTP code for the update, looks good otherwise

File: daemon/clients/pods.py (hunk 1 above)
Comment (deepankarm) [reply daemon/clients/pods.py]: You're right. Fixed

**Review by jacobowitz — APPROVED** [review]
:+1:

**Pull request merged**